rewrite-java/src/main/java/org/openrewrite/java/internal/parser/TypeTable.java

```diff
@@ -384,15 +402,14 @@ public void writeMethod(int access, String name, String descriptor,
                             parameterNames == null ? "" : String.join("|", parameterNames),
                             exceptions == null ? "" : String.join("|", exceptions)
                     );
+                    return true;
                 }
+                return false;
             }
 
-            public void writeField(int access, String name, String descriptor,
-                                   @Nullable String signature) {
-                if (((Opcodes.ACC_PRIVATE | Opcodes.ACC_SYNTHETIC) & access) == 0) {
```

Comment:
We're doing the same check in `writeMethod`, so figured we could remove it here for simplicity.